code-list-scripts/cypher/seasonal-adjustment.cypher

```diff
@@ -0,0 +1,7 @@
+CREATE CONSTRAINT ON (n:`_code_seasonal-adjustment`) ASSERT n.value IS UNIQUE;
+CREATE CONSTRAINT ON (n:`_code_list_seasonal-adjustment`) ASSERT n.value IS UNIQUE;
+CREATE (node:`_code_list`:`_code_list_seasonal-adjustment` { label:'seasonaladjustment', edition:'one-off' });
+MERGE (node:`_code`:`_code_seasonal-adjustment` { value:"nsa" });
```

Comment:
Did the business area provide the code 'nsa' and 'sa' or did we create them? if they're in our control, it would be better to spell it out in the code, but if its a standard the business are familiar with that might have to wait for another day